src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -0,0 +1,139 @@
+import torch
+from torch import nn
+
+from garage.torch.modules import GaussianMLPModule
+from garage.torch.policies import Policy
+
+class GaussianMLPPolicy(nn.Module, Policy):
```

Comment:
this class needs to expose its underlying distribution so that we can access differentiable paths through dist.entropy, dist.log_likelihood, etc.

Reply:
Now, users can get the distribution using forward function.
Do you mean the Policy should cache the distribution and return the information?

Reply:
thanks for pointing that out. let's try with the forward-only implementation for now.